src/test_build.py

```diff
@@ -0,0 +1,12 @@
+from docker import APIClient
```

Comment:
This will be removed. 
This is an extra file that I have added to work with the Docker APIS.